styles/issuesish-list-view.less

```diff
@@ -0,0 +1,65 @@
+@import "variables";
+
+// TODO: Rename
```

Comment:
🙈 Oh whoops, did I get the naming scheme wrong... ?

Reply:
Just a little. :wink: But no worries. It's more like a guide.

Typically, the double dash `--` is used for _variations_ of a component. Like `Button--big`, `Button--small`. They are both buttons, but maybe look a bit different. For `.github-Accordion-list--item` I think I'll change it to `.github-Accordion-listItem` or similar.

Reply:
Ohh right. Got it 👍